### PR TITLE
Add BBR testcases

### PR DESCRIPTION
**Pull request by Gfrom2016**
### Description of PR

After the fix in https://github.com/sonic-net/sonic-buildimage/pull/19590, add a new test case to test the bbr behavior after config reload.
Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [x] 202305
- [x] 202311
- [x] 202405

#### What is the motivation for this PR?
Add a new case to verify the BBR initialized behavior.
#### How did you do it?
Add the following new case under test_bgp_bbr.py
test_bbr_status_consistent_after_reload

#### How did you verify/test it?
https://dev.azure.com/mssonic/internal/_build/results?buildId=619063&view=results

### CODE REVIEW

File: tests/bgp/test_bgp_bbr.py [hunk 1]
```diff
@@ -153,23 +154,41 @@ def config_bbr_enabled(duthosts, setup, rand_one_dut_hostname, restore_bbr_defau
...
+
+
 @pytest.fixture(scope='module')
 def setup(duthosts, rand_one_dut_hostname, tbinfo, nbrhosts):
     duthost = duthosts[rand_one_dut_hostname]
 
     constants_stat = duthost.stat(path=CONSTANTS_FILE)
     if not constants_stat['stat']['exists']:
-        pytest.skip('No file {} on DUT, BBR is not supported')
+        pytest.skip('No constants.yml file on DUT, BBR is not supported')
```
Comment (ZhaohuiS) [inline tests/bgp/test_bgp_bbr.py:187]: If there is no constants.yml, is it a correct status? In my understanding, constants.yml should store many constans, right? If the file doesn't exist, it is in fatal condition, not just BBR is not supported, we have to fail the case, not just skip it, is it correct?

Comment (Gfrom2016) [reply tests/bgp/test_bgp_bbr.py:187]: I don't think constants file not exist should fail the case, it's not the bbr case's responsibility to make sure the file exists.

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Comment by StormLiangMS** [comment]
hi @ZhaohuiS any concern to approve?

**Review by ZhaohuiS — APPROVED** [review]
LGTM

**Pull request merged**